src/main/java/ratpack/zipkin/internal/RatpackClientRequestAdapter.java

```diff
@@ -65,7 +66,7 @@ public void addSpanIdToRequest(@Nullable final SpanId spanId) {
 
   @Override
   public Collection<KeyValueAnnotation> requestAnnotations() {
-    return Collections.singletonList(KeyValueAnnotation.create("http.uri", requestSpec.getUri().toString()));
+    return Collections.singletonList(KeyValueAnnotation.create(TraceKeys.HTTP_URL, requestSpec.getUri().toString()));
```

Comment:
@llinder given what we were discussing re. path vs. full URL - do you think we should go with path here as well?